solr/modules/s3-repository/src/test/org/apache/solr/s3/S3BackupRepositoryTest.java

```diff
@@ -246,7 +266,7 @@ private void doRandomAccessTest(String content, int position) throws Exception {
 
     try (S3BackupRepository repo = getRepository()) {
       // Open an index input on a file
-      pushObject("/my-repo/content", content);
+      pushObject("my-repo/content", content);
```

Comment:
@HoustonPutman I recall you fighting with S3 and Mock-S3 and how it handled directories...

Reply:
something changed with the version change of the mock lib